applications/HDF5Application/tests/test_hdf5_model_part_io.py

```diff
@@ -104,17 +104,25 @@ def _initialize_model_part(self, model_part):
             node.Set(STRUCTURE, bool(random.randint(-100, 100) % 2))
 
         for element in model_part.Elements:
-            element.SetValue(ACCELERATION, Vector([random.random(), random.random(), random.random()]))
+            if (element.Id % 2 == 0):
+                element.SetValue(DISPLACEMENT, Vector([random.random(), random.random(), random.random()]))
+            element.SetValue(VELOCITY, Vector([random.random(), random.random(), random.random()]))
+            if (element.Id % 3 == 0):
```

Comment:
I know randoms have been used in these tests before, but I really dislike the idea of unreproducible tests.